imperative/python/megengine/functional/elemwise.py

```diff
@@ -132,7 +132,36 @@ def sub(x, y):
 
 
 def mul(x, y):
-    r"""Element-wise `multiplication`."""
+    r"""Returns the tensor representing the element-wise multiplication of the two input tensors(also known as Hadamard product).
+
+    Args:
+        x: input tensor.
+        y: input tensor.
+
+    Returns:
+        output tensor.
```

Comment:
```suggestion
        an tensor containing the element-wise products. The returned array must have a data type determined by :ref:`dtype-promotion`.
```

这里的 :ref:`dtype-promotion` 在生成文档时会指向：

https://megengine.org.cn/doc/stable/zh/user-guide/model-development/tensor/dtype.html#dtype-promotion